homeassistant/components/homee/helpers.py

```diff
@@ -0,0 +1,25 @@
+"""Helper functions for the homee custom component."""
+
+import logging
+
+from pyHomee import Homee
+from pyHomee.model import HomeeNode
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_imported_nodes(config_entry) -> list[HomeeNode]:
+    """Get a list of nodes that should be imported."""
+    homee: Homee = config_entry.runtime_data
+    return homee.nodes
+
+
+def get_name_for_enum(att_class, att_id) -> str:
```

Comment:
Is this used?

Reply:
Yes, [here](https://github.com/Taraman17/ha-core/blob/e60d62df15ab9e7f317ebf64b1983c0a07cc2897/homeassistant/components/homee/entity.py#L39)